apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -272,9 +272,13 @@ function initializeBlocklyWrapper(blocklyInstance) {
     this.removeUnusedBlockFrame();
   };
 
+  // overridden dispose function definition at https://github.com/google/blockly/blob/develop/core/block_svg.ts#L863
```

Comment:
Nit for next time -- copy a permalink (one of the options when you click on a line on github), so as these files change, we have a consistent reference to the logic you're referencing.

Reply:
Oh interesting! Will definitely use a permalink next time - didn't know about this option. Thanks!